### ISSUE TITLE
base should be installed before cmd3

### ISSUE THREAD

**Issue by badmutex** [cloudmesh/get#13]
Command:

```
curl https://raw.githubusercontent.com/cloudmesh/get/master/cloudmesh/ubuntu/14.04.sh | bash
```

Output:

```
[SNIP]
+ pip install --upgrade pip
Downloading/unpacking pip from https://pypi.python.org/packages/py2.py3/p/pip/pip-7.1.0-py2.py3-none-any.whl#md5=b108384a762825ec20345bb9b5b7209f
  Downloading pip-7.1.0-py2.py3-none-any.whl (1.1MB): 1.1MB downloaded
Installing collected packages: pip
  Found existing installation: pip 1.5.4
    Uninstalling pip:
      Successfully uninstalled pip
Successfully installed pip
Cleaning up...
+ pip install cmd3==1.8.1
/home/ubuntu/ENV/local/lib/python2.7/site-packages/pip/_vendor/requests/packages/urllib3/util/ssl_.py:90: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.
  InsecurePlatformWarning
Collecting cmd3==1.8.1
/home/ubuntu/ENV/local/lib/python2.7/site-packages/pip/_vendor/requests/packages/urllib3/util/ssl_.py:90: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.
  InsecurePlatformWarning
  Downloading cmd3-1.8.1.tar.bz2
    Complete output from command python setup.py egg_info:
    Collecting cloudmesh-base
    /home/ubuntu/ENV/local/lib/python2.7/site-packages/pip/_vendor/requests/packages/urllib3/util/ssl_.py:90: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.
      InsecurePlatformWarning
      Downloading cloudmesh_base-3.2.0.tar.bz2
        Complete output from command python setup.py egg_info:
        Traceback (most recent call last):
          File "<string>", line 20, in <module>
          File "/tmp/pip-build-7teNj_/cloudmesh-base/setup.py", line 18, in <module>
            version = get_version_from_git()
          File "cloudmesh_base/setup.py", line 20, in get_version_from_git
            return Shell.git('tag').split("\n")[-1]
          File "cloudmesh_base/Shell.py", line 61, in git
            def git(cls, *args): return cls.execute('git',args)
          File "cloudmesh_base/Shell.py", line 271, in execute
            result = subprocess.check_output(os_command, stderr=subprocess.STDOUT).rstrip()
          File "/usr/lib/python2.7/subprocess.py", line 573, in check_output
            raise CalledProcessError(retcode, cmd, output=output)
        subprocess.CalledProcessError: Command '['git', 'tag']' returned non-zero exit status 128

        ----------------------------------------
    Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-7teNj_/cloudmesh-base
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "/tmp/pip-build-PAQ7MD/cmd3/setup.py", line 32, in <module>
        from cloudmesh_base.util import banner
    ImportError: No module named cloudmesh_base.util

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-PAQ7MD/cmd3
```

**Comment by laszewsk** [comment]
> script 14.04.sh is wrong

the coudmesh_base pip install must be before the cmd3 pip install. 

In final version may not matter as cmd3 implicitly installs cloudmesh_base, but possibly in the intermediate version that is used here.

**Comment by badmutex** [comment]
This is fixed in b33f2ba.
If the current run succeeds I'll merge into master and close the issue.

**Comment by rajaramcomputers** [comment]
Still the problem is prevailing in the latest download... Suggestions please...
